Sources/MessagingPush/PushHandling/ManualPushHandling.swift

```diff
@@ -4,6 +4,12 @@ import Foundation
 import UserNotifications
```

Comment:
I renamed this file and moved it into the `PushHandling` folder. The new file name and the comments I added to the top of the file are meant to better describe that these functions are separate from the automatic push click handling feature.